workers/unity/Packages/com.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -74,6 +77,9 @@ private static void Generate()
                 var schemaCompilerPath =
                     Path.GetFullPath(Path.Combine(Application.dataPath, SchemaCompilerRelativePath));
 
+                var workerJsonPath =
```

Comment:
This relative path ".." constant is only used once, why not just inline it?

Reply:
mainly to make it easier later on to change it, but I guess we need to rewrite parts of this class anyway as soon as we use FPL